src/celengine/category.cpp

```diff
@@ -109,11 +113,11 @@ void UserCategory::cleanup()
 UserCategory::CategoryMap UserCategory::m_allcats;
 UserCategory::CategorySet UserCategory::m_roots;
 
-UserCategory *UserCategory::newCategory(const std::string &s, UserCategory *p)
+UserCategory *UserCategory::newCategory(const std::string &s, UserCategory *p, const std::string &d)
```

Comment:
still just d

Reply:
Ah, here... I assumed it was in a header file. But if so, why `s` and `p` are ok?

Reply:
not ok actually :) we should avoid cryptic names, but just to keep the pr as small as possible i said only about the new argument